Frontend/src/app/app.module.ts

```diff
@@ -16,10 +16,20 @@ import {NewestPricesListComponent} from './components/newest-prices-list/newest-
 import {FormsModule} from '@angular/forms';
 import {PageNotFoundPageComponent} from './pages/page-not-found-page/page-not-found-page.component';
 import {MatMenuModule} from '@angular/material/menu';
-import { BrowserAnimationsModule } from '@angular/platform-browser/animations';
+import {BrowserAnimationsModule} from '@angular/platform-browser/animations';
 import {ImprintComponent} from './pages/imprint/imprint.component';
 import {PrivacyComponent} from './pages/privacy/privacy.component';
 import {NgcCookieConsentModule, NgcCookieConsentConfig} from 'ngx-cookieconsent';
+import {MatSlideToggleModule} from '@angular/material/slide-toggle';
+import {TopBarComponent} from './components/top-bar/top-bar.component';
+import {RouterModule} from '@angular/router';
+import {MatButtonModule} from '@angular/material/button';
```

Comment:
Codacy has a fix for the issue: [Strings must use doublequote.](https://app.codacy.com/gh/Mueller-Patrick/Betterzon/pullRequest?prid=7378189)
```suggestion
import {MatButtonModule} from "@angular/material/button";
```